README.md

```diff
@@ -347,3 +347,17 @@ recent and largely contained to Envoy, which already supports B3 single format.
 Sending a larger second header would increase the pain to others who don't have
 this problem anyway. Put another way, the few proxies missing B3 single header
 support should update instead of burdening more people from lack thereof.
+
+# Showcase: propagation from an Apache reverse proxy
+
+Zipkin users were <a href="https://github.com/openzipkin/zipkin/issues/1479">wondering</a> 
+if it is possible to start B3 propagation from an Apache reverse proxy. @adammichalik 
+came up with a <a href="https://github.com/openzipkin/zipkin/issues/1479#issuecomment-682639979">clever</a> 
+way to accomplish this, using Apache modules only:
+
+```
+RequestHeader setifempty X-B3-TraceId "expr=%{md5:%{reqenv:UNIQUE_ID}}"
+RequestHeader edit X-B3-TraceId "^(.{0,16}).*$" "$1"
```

Comment:
this is incorrect, it should be the last 16 chars, not the first. Also `X-B3-SpanId` should be set to the last 16 chars.

Doing this without setting the sampling flag will result in the first hop sampling, which is probably ok as it could be tricky to put sampling logic here :D.

OTOH an example of `b3: 0` for /health path would be nice as it shows how to stop things from tracing that.

Reply:
@adriancole Can you please explain a bit on why "this is incorrect, it should be the last 16 chars, not the first"? The way I was thinking (but I wasn't thinking too much of it) is that MD5 generates a "uniformly random" hex string of 32 chars, so it doesn't matter if we take the first, middle or last 16 chars - as a matter of fact this trick was simply a way to generate a pseudorandom 16 char hex string.

Reply:
sure it is here. https://github.com/openzipkin/b3-propagation/blob/master/STATUS.md and also explained here https://github.com/openzipkin/zipkin/blob/master/zipkin/src/main/java/zipkin2/storage/StorageComponent.java#L106-L144

Seems yeah this should be in the top-level readme :D Offers welcome to fix that!

The main thing is that even if at your site, you feel truncating to 64bit is a constant, we shouldn't hint here picking the left hand side as that will conflict with other practice. I think many would use this with 128-bit IDs as almost all libraries at least truncate back on unsupported for the last several years. It isn't necessary to do that here, but it is interesting to show the option. When showing the option, we should be specific on how.

Reply:
Fair points, thanks for the explanation.

Reply:
no prob!